views/components/Sidebar/SidebarLink.jsx

```diff
@@ -0,0 +1,17 @@
+import { forwardRef } from 'react'
+import { NavLink } from 'react-router-dom'
+import './SidebarLink.css'
+
+const SidebarLink = forwardRef((props, ref) => (
+  <NavLink
+    ref={ref}
+    to={props.to}
+    className={({ isActive }) =>
+      isActive ? 'SidebarLink_link-active' : 'SidebarLink_link'
+    }
```

Comment:
```suggestion
    className={({ isActive }) => classnames('SidebarLink_link', { 'SidebarLink_link-active': isActive })}
```

Reply:
we don't have to use classNames this is a built in function from NavLink react router it isn't the React classNames

Reply:
classnames is just a utility for conditionally adding classes. We don't have to use it, but I _think_ we want the `SidebarLink_link` class applied in both conditions (so when not active the classes are `SidebarLink_link` and when active the classes are `SidebarLink_link SidebarLink_link-active`).

Reply:
gotcha, we don't need it in this case

Reply:
right, because we're duplicating styles in both classes. We don't need to solve this right now but ideally the active modifier class only applies the styles that are specific to that modification and the other class has the shared styles.